### PR TITLE
feat:documentation site with components

### PR DESCRIPTION
**Pull request by TheAmlanSaikia**
The components are reviewed. Only made the documentation site now with the components. Just need to review the Documentation.html structure and Documentation.css. Not fully responsive.
documentation site version 1

### CODE REVIEW

File: Documentation/Documentation.css [hunk 1]
```diff
@@ -0,0 +1,145 @@
...
+
+.navbar {
+    grid-area: navbar;
+    min-height: 11vh;
+    background-color: var(--pure-white);
+    box-shadow: var(--box-shadow-dark);
+    padding: var(--value-2) var(--value-2);
+    top: 0;
+    z-index: 2;
+    /* gap: var(--value-2); */
```
Comment (Aditi0505) [inline Documentation/Documentation.css:29]: Can remove commented code if not needed.

File: Documentation/Documentation.css [hunk 2]
```diff
@@ -0,0 +1,145 @@
...
+
+.sidebar-links:hover {
+    display: block;
+    text-decoration: underline;
+    text-decoration-thickness: 2.5px;
+    text-decoration-color: var(--warning-text-color);
+}
+
+
+
```
Comment (Aditi0505) [inline Documentation/Documentation.css:83]: Can remove extra spacing.

File: Documentation/Documentation.css [hunk 3]
```diff
@@ -0,0 +1,145 @@
...
+
+
+/* 
+@media screen and (max-width: 800px){
+    *{
+        font-size: 60%;
+    }
```
Comment (Aditi0505) [inline Documentation/Documentation.css:144]: Can remove commented code if not needed.

File: Documentation/Documentation.html [hunk 4]
```diff
@@ -0,0 +1,1108 @@
...
+                        <div class="component-box">
+                            <div class="component-display">
+                                <div class="flex-row justify-start items-center flex-wrap m-2">
+                                    <a class="btn btn-secondary-solid">Solid</a>
+                                    <a class="btn btn-secondary-outline">Outline</a>
+                                    <a class="underline-btn btn-secondary-underline">Text-Link</a>
+                                    <a href="link" class="btn link-btn btn-secondary-link">Link</a>
+                                    <a class="btn btn-icon-secondary"><span class="btn-icon"><i
+                                                class="fas fa-trash"></i></i></span></a>
+                                </div>
```
Comment (Aditi0505) [inline Documentation/Documentation.html]: Can use code formatting tool here.

File: Documentation/Documentation.html [hunk 5]
```diff
@@ -0,0 +1,1108 @@
...
+                                    sandbox="allow-scripts allow-same-origin">
+                                </iframe>
+                            </div>
+                        </div>
+                </article>
+
+
+
+
```
Comment (Aditi0505) [inline Documentation/Documentation.html]: Can remove extra spacing.

File: Documentation/Documentation.html [hunk 6]
```diff
@@ -0,0 +1,1108 @@
...
```
Comment (Aditi0505) [inline Documentation/Documentation.html]: Can remove extra spacing here.

File: Documentation/Documentation.html [hunk 7]
```diff
@@ -0,0 +1,1108 @@
...
```
Comment (Aditi0505) [inline Documentation/Documentation.html]: Can remove extra spacing here

**Review by Aditi0505 — APPROVED** [review]
Left few suggestion. Rest looks good to me.

**Review by ranitasahaa1230 — APPROVED** [review]
Looks good to merge!

**Pull request merged**